examples/cloud-tomee-azure/README.adoc

```diff
@@ -0,0 +1,101 @@
+
+== Azure Setup
+
+In order for the Azure plugin to work, you will need to have an Azure account and add a subscription to it.
+Then, on yor development machine, install the Azure command-line interface (CLI) and authenticate with the command
+line, before you can deploy your application.
+
+- Create an Azure Account, if you don't have one, at https://azure.microsoft.com/en-us
+- Use the free option, if available or https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade[add a subscription].
+- https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Install] the Azure (CLI) according
+to operating system of the computer you are using to develop.
+
+- Finally you can setup your development computer.
+
+=== Login into Azure
+
+`az login`
+
+The result:
+
+----
+[
+   {
+     "cloudName": "AzureCloud",
+     "id": "aaaaaaaa-aaaa-aaaa-aaaaa-aaaaaaaaaa",
+     "isDefault": true,
+     "name": "Pay-As-You-Go",
+     "state": "Enabled",
+     "tenantId": "bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb",
+     "user": {
+       "name": "<your azure account's email>",
+       "type": "user"
+     }
+   }
+ ]
+----
+The TenantId is someone that can register and manage apps, yourself. You will need that for later.
+
+
+=== Create a service principal
+An Azure service principal is a security identity used by user-created apps, services,
+and automation tools to access specific Azure resources:
+
+`az ad sp create-for-rbac --name ServicePrincipalName --password <Choose a strong password>`
+
+----
+az ad sp create-for-rbac --name  http://cloud-tomee-azure --password <password for this app>
+
+{
+  "appId": "cccccccc-cccc-cccc-cccc-ccccccccccccccc",
+  "displayName": "cloud-tomee-azure",
+  "name": "http://cloud-tomee-azure",
+  "password": "<password for this app>",
+  "tenant": "bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb"
+}
+----
+The ServicePrincipalName will also be set as subdomain name. In this case "http://cloud-tomee-azure".
+The appId is the identification of the app service.
+
+=== Configure Maven
+
+You could continue just using the Azure CLI for authentication but we can also do it permanently with maven.
+
+In that case we need to edit Maven's settings.xml so the azure-webapp-maven-plugin can authenticate against Azure:
+
+You can add a new server to `~/.m2/settings.xml` like this:
+----
+<server>
+  <id>azure-auth</id>
+  <configuration>
+     <client>cccccccc-cccc-cccc-cccc-ccccccccccccccc</client>
+     <tenant>bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb</tenant>
+     <key><password for this app></key>
+     <environment>AZURE</environment>
+   </configuration>
+</server>
+----
+That's it. You can now build the example and deploy it to Azure just using Maven:
+
+`mvn clean install -Pazure-single-jar azure-webapp:deploy`
+
+The azure-webapp is explicitly invoked because it relies on you Azure account. The standard TomEE build will not use an Azure account.
+
```

Comment:
I forgot to mention that the readme can include the reference to the azur Ticket and the hack one needs to do about the JAVA_HOME=/usr/lib/jvm/zulu-8-azure-jre-headless_8.33.0.1-8.0.192-linux_musl_x64 in the application settings.

Reply:
done